README.md

```diff
@@ -1,88 +1,163 @@
 
-To verify the command runs correctly. You should see the available paramaters that can be used.
+- `output_file_path`: Path where the result NC file will be saved
+- `erddap_configuration_file` (optional): Path to connection configuration file
 
-  - The first argument is the path to the source Gird file (There an example in this repo)
-  - The second argument is the path to the osom NC file.
-  - Third argument is the path to the output_folder of the raw file
-  - Fourth argument is the attribute in the nc file to be analyzed
-  - (Optional) Fifth argument is a list of time frames to be analyzed (by default analyzes the whole dataset)
-  
- ### Example
+A configuration file is a `.json` that looks like this:
 
-`poetry run osom-converter path-to-osom-grid-file\osom_grid4_mindep_smlp_mod7.nc path-to-osom-nc-file\ocean_his_0196.nc path-to-output-folder temp`
-  
-### Example
+```json
+{
+  "erddap_connection": {
+    "server": "https://pricaimcit.services.brown.edu/erddap",
+    "protocol": "griddap",
+    "dataset_id": "model_data_57db_4a85_81d9"
+  }, 
+  "erddap_constrainsts": {
+        "variables":["SalinityBottom","SalinitySurface","WaterTempSurface","WaterTempBottom"],  
+        "time>=": "2019-12-30T00:00:00Z",
+        "time<=": "2019-12-31T00:00:00Z",
+        "time_step": 1,
+        "eta_rho>=": 0,
+        "eta_rho<=": 1099,
+        "eta_rho_step": 1,
+        "xi_rho>=": 0,
+        "xi_rho<=": 999,
+        "xi_rho_step": 1
+  }
+}
+```
+
+Modify the imporant attributes such as `server ULR`, `dataset_id` and `contrainst` according to the data you want to visualize.
 
-`poetry run osom-converter path-to\nc-file.raw path-to-result-folder`
+There is a default `erddap_configuration.json` file located in the `configuration` folder.
 
-# For developers
+This is an example on how to run the command:
+
+`erddap_query path_to_result_folder path_to_configuration_file.json`
+
+### Converting data from NC to .raw or .png
+
+The command `osom-converter` has only one parameter:
+
+- `parameters_file_path`: Path to a `.json` file that looks like this:
+
+```json
+{
+  "osom_grid_file": "path_to_grid_file/osom_grid4_mindep_smlp_mod7-netcdf4.nc",
+  "osom_data_file": "path_to_nc_datafile/model_data_57db_4a85_81d9.nc",
+  "output_folder": "path_to_output_folder",
+  "data_descriptor" : "WaterTempSurface",
+  "time_frames": [0],
+  "layer" : "surface",
+  "to_texture_atlas": true
+}
+```
+
+- There is an example grid file located in the `resources\grid-data` folder
+- The `osom_data_file` is the resulting osom-nc file fomr the `erddap_query` command.
+- `data_descriptor` is one of the variables in the erddap data base.
+- When the osom-nc files respresent multiple time frames ( i.e data representing different hours in a day ), the data is store in an 1D-array. `time_frames` argument specify the time frame you want to convert into volume viewer data. By default, it will convert all the available time frames.
```

Comment:
```suggestion
- When the osom-nc files respresent multiple time frames ( i.e data representing different hours in a day ), the data is stored in an 1D-array. The `time_frames` argument specify the time frame you want to convert into volume viewer data. By default, it will convert all the available time frames.
```